cilium/egress-connection-timeouts.md

```diff
@@ -0,0 +1,313 @@
+# CFP-TBD: Egress Connection Timeouts
+
+**SIG: SIG-DATAPATH, EGRESS-GATEWAY**
+
+**Begin Design Discussion:** 2024-01-16
+
+**Cilium Release:** X.XX
```

Comment:
Small administrative item now that the PR is approved: v1.18 development closes soon, so v1.19 would be the next ideal target.
```suggestion
**Cilium Release:** 1.19
```